tfjs-backend-webgpu/src/conv_backprop_webgpu.ts

```diff
@@ -97,3 +97,71 @@ export class Conv2DDerInputProgram implements WebGPUProgram {
   `;
   }
 }
+
+export class Conv2DDerFilterProgram implements WebGPUProgram {
+  variableNames = ['x', 'dy'];
+  uniforms =
+      'pads : vec2<i32>, stride : vec2<i32>, batchSize : i32, outHeight : i32, outWidth : i32, inHeight : i32, inWidth : i32,';
```

Comment:
Why not use padTop and padLeft directly so that their usages are more easy to read (rather than pads[0] and pads[1])?
The names of pad and stride are not consistent. Maybe pads -> pad?